src/current/v25.1/set-up-logical-data-replication.md

```diff
@@ -85,87 +96,148 @@ When you run LDR in [`immediate` mode](#modes), you cannot replicate a table wit
 
-    Once LDR has started, an LDR job will start on the destination cluster. You can [pause]({% link {{ page.version.version }}/pause-job.md %}), [resume]({% link {{ page.version.version }}/resume-job.md %}), or [cancel]({% link {{ page.version.version }}/cancel-job.md %}) the LDR job with the job ID. Use [`SHOW LOGICAL REPLICATION JOBS`]({% link {{ page.version.version }}/show-logical-replication-jobs.md %}) to display the LDR job IDs:
+    Run the following from the **destination** cluster (i.e, the cluster that does not have the table currently):
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
-    SHOW LOGICAL REPLICATION JOBS;
-    ~~~
-    ~~~
-            job_id        | status  |          targets          | replicated_time
-    ----------------------+---------+---------------------------+------------------
-    1012877040439033857   | running | {database.public.table}   | NULL
-    (1 row)
+    CREATE LOGICALLY REPLICATED TABLE {database.public.destination_table_name} FROM TABLE {database.public.source_table_name} ON 'external://source' WITH bidirectional ON 'external://destination';
     ~~~
 
-    If you're setting up bidirectional LDR, both clusters will have a history retention job and an LDR job running.
+You can include multiple tables in the LDR stream for unidirectional or bidirectional setups. Ensure that the table name in the source table list and destination table list are in the same order:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICALLY REPLICATED TABLE ({database.public.destination_table_name_1}, {database.public.destination_table_name_2}) FROM TABLE ({database.public.source_table_name_1}, {database.public.source_table_name_2}) ON 'external://source' WITH bidirectional ON 'external://destination', label=track_job;
+~~~
+
+With the LDR streams created, move to [Step 4](#step-4-manage-and-monitor-the-ldr-jobs) to manage and monitor the jobs.
+
+#### `CREATE LOGICAL REPLICATION STREAM`
+
+Ensure you've created the table on the destination cluster with a matching schema definition to the source cluster table. From the **destination** cluster, start LDR. Use the fully qualified table name for the source and destination tables:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.source_table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.destination_table_name};
+~~~
+
+You can change the default `mode` using the `WITH mode = validated` syntax.
+
+If you would like to add multiple tables to the LDR job, ensure that the table name in the source table list and destination table list are in the same order:
```

Comment:
oh yeah this was it, can we say why this is necessary? as a user, it feels a bit fragile to say "hold it exactly like this!!!!!" but not say why, like what if i generate lists and shuffle them will it be slow or just not work?